src/core/Table/filters/FilterToggle.tsx

```diff
@@ -49,12 +49,16 @@ export const FilterToggle = <T extends Record<string, unknown>>(
           placement='bottom'
           visible={isVisible}
           onClickOutside={close}
+          appendTo={document.body}
```

Comment:
Fix for https://github.com/iTwin/iTwinUI-react/issues/69. Otherwise filter would be as a child of a header column which handles clicks and pointer cursor for sorting.

Reply:
Is that necessary?

I would think the `stopPropagation` should be enough. Do we need both?

Reply:
Without this line:
![filter-sort](https://user-images.githubusercontent.com/36186912/118678310-93443480-b805-11eb-9bc3-19ddc5c8649f.gif)

Reply:
Basefilter takes care of stopping the event from going to the header
The only issue that remains is the pointer icon when hovering on the side. 
I ignored it on my PR because it is just a UI Issue and not affecting the sorting 
Correct fix would be to fix the css for the header

Reply:
@basanta078  It's not just the pointer. Notice the table header getting highlighted. That's because the filter dropdown is located inside the header div.

CSS will not help here, so it needs to be in your PR or in this one.

Reply:
Yeah, CSS won't help because without this line filter is rendered as a header child.

Reply:
sounds good, I added that fix to the other PR